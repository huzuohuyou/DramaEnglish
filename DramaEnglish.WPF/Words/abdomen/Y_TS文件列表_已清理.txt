http://ssv-video.agilestudio.cn/null_26973198_6_15ae34/null_26973198_6_15ae34.m3u8
https://ssv-api.agilestudio.cn/api/video/vtt-subtitle/9991-563000-595000.vtt
http://ssv-video.agilestudio.cn/null_26973198_6_15ae34/null_26973198_6_15ae3456.ts
