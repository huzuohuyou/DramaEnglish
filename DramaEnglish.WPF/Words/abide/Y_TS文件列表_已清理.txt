https://ssv-api.agilestudio.cn/api/dict/video-type-options?_platform=web&_versioin=0.2.5&_ts=1627642057103
https://ssv-api.agilestudio.cn/api/dict/lang-options?_platform=web&_versioin=0.2.5&_ts=1627642057102
https://ssv-api.agilestudio.cn/api/dict/video-category-options?_platform=web&_versioin=0.2.5&_ts=1627642057103
https://ssv-api.agilestudio.cn/api/search?subtitleKeyword=abide&exactFlag=0&phraseMode=0&page=1&limit=20&_platform=web&_versioin=0.2.5&_ts=1627642057103
https://ssv-api.agilestudio.cn/api/message/unread-count?_platform=web&_versioin=0.2.5&_ts=1627642057109
http://ssv-video.agilestudio.cn/9829_f7ea1a/9829_f7ea1a.m3u8
https://ssv-api.agilestudio.cn/api/video/vtt-subtitle/9829-1894000-1925000.vtt
http://ssv-video.agilestudio.cn/9829_f7ea1a/9829_f7ea1a189.ts
http://ssv-video.agilestudio.cn/9829_f7ea1a/9829_f7ea1a190.ts
