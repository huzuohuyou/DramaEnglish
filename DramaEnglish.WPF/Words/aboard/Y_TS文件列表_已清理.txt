https://ssv-api.agilestudio.cn/api/dict/video-type-options?_platform=web&_versioin=0.2.5&_ts=1627641390598
https://ssv-api.agilestudio.cn/api/dict/lang-options?_platform=web&_versioin=0.2.5&_ts=1627641390598
https://ssv-api.agilestudio.cn/api/search?subtitleKeyword=aboard&exactFlag=0&phraseMode=0&page=1&limit=20&_platform=web&_versioin=0.2.5&_ts=1627641390599
https://ssv-api.agilestudio.cn/api/dict/video-category-options?_platform=web&_versioin=0.2.5&_ts=1627641390599
https://ssv-api.agilestudio.cn/api/message/unread-count?_platform=web&_versioin=0.2.5&_ts=1627641390601
http://ssv-video.agilestudio.cn/4599_7db517/4599_7db517.m3u8
https://ssv-api.agilestudio.cn/api/video/vtt-subtitle/4599-8749000-8783000.vtt
http://ssv-video.agilestudio.cn/4599_7db517/4599_7db517-00885.ts
