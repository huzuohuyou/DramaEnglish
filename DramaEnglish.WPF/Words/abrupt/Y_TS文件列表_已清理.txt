http://ssv-video.agilestudio.cn/4842_a4cb41/4842_a4cb41.m3u8
https://ssv-api.agilestudio.cn/api/video/vtt-subtitle/4842-1995000-2027000.vtt
http://ssv-video.agilestudio.cn/4842_a4cb41/4842_a4cb41-00202.ts
