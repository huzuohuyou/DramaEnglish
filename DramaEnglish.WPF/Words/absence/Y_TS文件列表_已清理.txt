http://ssv-video.agilestudio.cn/143_26957989_10_82d3ce/143_26957989_10_82d3ce.m3u8
https://ssv-api.agilestudio.cn/api/video/vtt-subtitle/3759-850000-881000.vtt
http://ssv-video.agilestudio.cn/143_26957989_10_82d3ce/143_26957989_10_82d3ce-00086.ts
http://ssv-video.agilestudio.cn/143_26957989_10_82d3ce/143_26957989_10_82d3ce-00087.ts
