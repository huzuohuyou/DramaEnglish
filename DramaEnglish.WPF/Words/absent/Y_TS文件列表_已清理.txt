http://ssv-video.agilestudio.cn/9691_a01963/9691_a01963.m3u8
https://ssv-api.agilestudio.cn/api/video/vtt-subtitle/9691-1882000-1914000.vtt
http://ssv-video.agilestudio.cn/9691_a01963/9691_a01963187.ts
http://ssv-video.agilestudio.cn/9691_a01963/9691_a01963189.ts
