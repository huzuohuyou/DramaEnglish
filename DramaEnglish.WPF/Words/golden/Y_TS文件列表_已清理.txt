http://ssv-video.agilestudio.cn/171_3754368_20_296460/171_3754368_20_296460.m3u8
https://ssv-api.agilestudio.cn/api/video/vtt-subtitle/4113-125000-156000.vtt
http://ssv-video.agilestudio.cn/171_3754368_20_296460/171_3754368_20_296460-00014.ts
http://ssv-video.agilestudio.cn/171_3754368_20_296460/171_3754368_20_296460-00015.ts
http://ssv-video.agilestudio.cn/171_3754368_20_296460/171_3754368_20_296460-00016.ts
